documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,422 @@
+Getting Started - Generate the RLC rest-level client libraries.
```

Comment:
Can we also rename the file to be RLC-quickstart and any reference to LLC make it RLC 😄

Reply:
Updated.